packages/squiggle-lang/benchmark/generators.ts

```diff
@@ -0,0 +1,48 @@
+export let generateFloatRange = (min: number, max: number): number =>
+  Math.random() * (max - min) + min;
+export let generateIntRange = (min: number, max: number): number =>
+  Math.floor(generateFloatRange(min, max));
+
+export let generateIntMin = (min: number): number => generateIntRange(min, 100);
+
+export let generateInt = (): number => generateIntMin(-100);
```

Comment:
why max of 100 and min of -100?

Reply:
No particular reason. Would you prefer int max and int min?

Reply:
Yea. I assume it would be better to be as wide as possible, though if it starts crashing, then we can constrain it.